xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -167,8 +176,8 @@ This constructor is protected. It can only be called from the <xref:System.Runti
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Runtime.Loader.AssemblyLoadContext&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Returns an IEnumerable for all <see cref="T:System.Runtime.Loader.AssemblyLoadContext" /> instances.</summary>
```

Comment:
IEnumerable should be marked as code (or maybe even a reference to the docs) - several times below as well.

Reply:
I ended up reverting this because the links are already automatically generated.  Switched `IEnumerable` to the generic term `collection`.